simple_lru_cache/simple_lru_cache.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2016 Google Inc. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+// For inclusion in .h files. The real class definition is in
+// simple_lru_cache_inl.h.
+
+#pragma once
+
+#include <functional>
+
+#include "absl/container/flat_hash_map.h"  // for hash<>
+
+namespace google {
+namespace jwt_verify {
```

Comment:
please change namespace from `jwt_verify` to `simple_lru_cache`  thanks

Reply:
Done.